textdb/textdb-dataflow/pom.xml

```diff
@@ -41,6 +41,21 @@
             <artifactId>textdb-storage</artifactId>
             <version>${textdb.version}</version>
         </dependency>
+
+
+        <dependency>
+            <groupId>edu.stanford.nlp</groupId>
+            <artifactId>stanford-corenlp</artifactId>
+            <version>3.6.0</version>
+        </dependency>
+        <dependency>
+            <groupId>edu.stanford.nlp</groupId>
+            <artifactId>stanford-corenlp</artifactId>
+            <version>3.6.0</version>
+            <classifier>models</classifier>
+        </dependency>
+
+
         <dependency>
             <groupId>org.mockito</groupId>
```

Comment:
I think my current test cases is good. If using Mockito, I would use it to mock the source operator and define the return value of getNextTuple(). It works just like the current one but we isolate the ScanBasedSourceOperator. In this case if someone modify the ScanBasedSourceOperator, the test case could remain unchanged.